scripts/pre_commit_linter.py

```diff
@@ -979,6 +984,24 @@ def _check_comments(all_files):
                         print '%s --> Line %s: %s' % (
                             filename, line_num + 1, message)
 
+                # Check that comment starts with a space.
```

Comment:
Perhaps explain the significance of `#!`, too.

Reply:
Done.